plugins/main/public/components/overview/mitre/intelligence/intelligence.tsx

```diff
@@ -93,19 +95,26 @@ export const ModuleMitreAttackIntelligence = compose(
   );
 
   useEffect(() => {
-    const urlParams = new URLSearchParams(location.href);
-    const redirectTab = urlParams.get('tabRedirect');
-    if (redirectTab) {
-      setSelectedResource(redirectTab);
+    if (!paramTabRedirect) {
+      NavigationService.getInstance().updateAndNavigateSearchParams({
+        tabRedirect: selectedResource,
+      });
     }
   }, []);
 
+  useEffect(() => {
+    setSelectedResource(paramTabRedirect);
+  }, [paramTabRedirect]);
```

Comment:
**suggestion**: this could be avoided if using the value of the URL query parameter directly that is provided by the `useRouterSearch` hook.

Reply:
This would need more development and care for possible bugs, and due to time issues it will be left for later.